main.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
I still think it should be in `server/` though, since it's a sample usage of this package.

Also, the `server/README.md` seems to describe it that way.